android/app/src/main/res/layout/item_data.xml

```diff
@@ -4,43 +4,44 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="?attr/selectableItemBackground">
+    android:background="?attr/selectableItemBackground"
+    android:padding="@dimen/default_margin_16"
+    tools:showIn="@layout/fragment_home">
 
-    <androidx.appcompat.widget.AppCompatImageView
-        android:id="@+id/ivDataThumbnail"
-        android:layout_width="100dp"
-        android:layout_height="0dp"
-        android:scaleType="centerCrop"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintHeight_min="100dp"
+    <ImageView
+        android:id="@+id/ivSmsBullet"
```

Comment:
Should we rename `item_data` to `item_sms` for consistency?
- `item_data` -> `ivData`
- `item_sms` -> `ivSms`

Reply:
Btw, I think we should add the label for the datetime.

Reply:
Let's do it later since the endpoint doesn't include this field for now. 🙏🏻